platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/isolated/IsolatedProjectsParallelConfigurationIntegrationTest.groovy

```diff
@@ -66,6 +65,18 @@ class IsolatedProjectsParallelConfigurationIntegrationTest extends AbstractIsola
         [":build", "--no-configure-on-demand"] | [":build"]
     }
 
+    def 'parallel configuration can be disabled in favor of configure-on-demand'() {
+        given:
+        server.expect("configure-root")
+        server.expect("configure-a")
```

Comment:
💭  The server here waits for some heuristic time (500ms?) to ensure no further requests are received. Though, I don't think it makes it clear that `:b` was not evaluated at all. I'd make it more explicit.

Ideally, we should be checking whether the project has been evaluated via build operations. But a println in `:b`'s build logic and the absence of it in the output should suffice.

Reply:
Printing doesn't allow to verify sequentiality of configuration tho

Reply:
My suggestion is only to add a validation, not remove the ones already present